iroh/src/node/builder.rs

```diff
@@ -239,19 +244,52 @@ impl<D: Map> Builder<D> {
             #[cfg(any(test, feature = "test-utils"))]
             insecure_skip_relay_cert_verify: false,
             gc_done_callback: None,
+            blob_events: MockEventSender,
+        }
+    }
+}
+
+impl<D, E> Builder<D, E>
+where
+    D: BaoStore,
+    E: iroh_blobs::provider::EventSender,
```

Comment:
I think the EventSender ist not important enough to warrant its own type parameter. Maybe we can make this an `Arc<dyn` or `Box<dyn` thing.